pytest_fixtures/component/provision_pxe.py

```diff
@@ -86,6 +89,31 @@ def module_provisioning_rhel_content(
     ).create()
     ak.add_subscriptions(data={'subscription_id': default_subscription.id})
 
+    # add sat client repo
```

Comment:
How much time does this new block add to the execution time? Just roughly.

I would say it is 2 min max.

Also, I have been thinking about whether this new block can be in a separate fixture. The assumption here is that not every test that uses `module_provisioning_rhel_content` needs a client repo to be present on the system.
On the other side, it is _just_ a small client repo.

Reply:
An extra fixture here makes sense to me. I would rather keep this fixture to minimum rhel content needed for provisioning.